### PR TITLE
add comment

### PR DESCRIPTION
**Pull request by Karol-Blazejowski-Eclipse**

### CODE REVIEW

File: src/main/java/pl/bushik/demo/DemoApplication.java [hunk 1]
```diff
@@ -3,11 +3,17 @@
...
+//**
+* Demo application
+**/
+
 @SpringBootApplication
 public class DemoApplication {
 
 	public static void main(String[] args) {
 		SpringApplication.run(DemoApplication.class, args);
+
```
Comment (Karol-Blazejowski-Eclipse) [inline src/main/java/pl/bushik/demo/DemoApplication.java]: this empty lines aren't necessary

File: src/main/java/pl/bushik/demo/DemoApplication.java [hunk 2]
```diff
@@ -3,11 +3,17 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+//**
+* Demo application
+**/
```
Comment (Karol-Blazejowski-Eclipse) [inline src/main/java/pl/bushik/demo/DemoApplication.java]: it is not a proper comment section, please fix that

File: src/main/java/pl/bushik/demo/DemoApplication.java [hunk 3]
```diff
@@ -3,11 +3,17 @@
...
+
 @SpringBootApplication
 public class DemoApplication {
```
Comment (Karol-Blazejowski-Eclipse) [inline src/main/java/pl/bushik/demo/DemoApplication.java:10]: test review

**Review by Karol-Blazejowski-Eclipse — COMMENTED** [review]
review - needs work

File: src/main/java/pl/bushik/demo/DemoApplication.java [hunk 4]
```diff
@@ -3,11 +3,17 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+//**
+* Demo application
```
Comment (bushik) [inline src/main/java/pl/bushik/demo/DemoApplication.java:7]: it is not a proper comment

**Review by bushik — CHANGES_REQUESTED** [review]
please fix problems

**Review by bushik — APPROVED** [review]

**Review by bushik — APPROVED** [review]

**Pull request merged**